tools/osbuilder/rootfs-builder/ubuntu/Dockerfile.in

```diff
@@ -14,18 +14,21 @@ RUN apt-get update && \
     ca-certificates \
     curl \
     g++ \
-    $(gcc_arch="@ARCH@" && [ "$(uname -m)" != "$gcc_arch" ] && ( \
+    "$(gcc_arch="@ARCH@" && [ "$(uname -m)" != "$gcc_arch" ] && ( \
```

Comment:
I got the following error with the double quotes:

```
E: Unable to locate package gcc-s390x-linux-gnu libc6-dev-s390x-cross
```

The build got successfully finished without them. I think they make 2 packages' names into 1. :wink: